src/main/resources/templates/layout.html

```diff
@@ -39,6 +39,9 @@
             <li class="nav-item">
                 <a class="nav-link" href="/mgmt/invites/send">Send Invite</a>
             </li>
+            <li class="nav-item">
+                <a class="nav-link" href="/mgmt/organisation">Organisation</a>
```

Comment:
Link label "Organisation" should be replaced with something like this: "Assign Reporting Permission"

Reply:
This is again under progress to decide labels on UI. Have notified to Mo and he will let me know once decided and will be changed accordingly.

Reply:
Yes, business should always decide upon website 'copy'.